memphis/memphis.py

```diff
@@ -420,6 +427,12 @@ async def producer(
                 raise MemphisError(create_res["error"])
 
             internal_station_name = get_internal_name(station_name)
+
+            if not create_res["partitions_update"]["partitions_list"] is None:
```

Comment:
Please change to `if create_res["partitions_update"]["partitions_list"] is not None`